app/src/main/java/com/kickstarter/viewmodels/ProjectViewModel.kt

```diff
@@ -834,13 +832,18 @@ interface ProjectViewModel {
                     .map { isPledgeCTA(it) }
                     .compose<Boolean>(takeWhen(this.nativeProjectActionButtonClicked))
 
-            currentFullProjectData
-                    .compose<Pair<ProjectData, User?>>(combineLatestPair(this.currentUser.observable()))
+            fullProjectDataAndCurrentUser
                     .compose<Pair<Pair<ProjectData, User?>, Boolean>>(combineLatestPair(shouldTrackCTAClickedEvent))
                     .filter { it.second }
                     .map { it.first }
                     .compose(bindToLifecycle())
                     .subscribe { this.optimizely.track(PROJECT_PAGE_PLEDGE_BUTTON_CLICKED, it.second, it.first.refTagFromIntent()) }
+
+            fullProjectDataAndCurrentUser
+                    .compose<Pair<ProjectData, User?>>(takeWhen(creatorInfoClicked))
+                    .filter { it.first.project().isLive && !it.first.project().isBacking }
```

Comment:
Do we not show the creator info if you've already backed the project? 🤔

Reply:
This is only for tracking. I'm only tracking this for live not backed projects. Otherwise, I think the experiment would have really low conversion because it would fire for ended and ? But not sure because the experiment process has been pretty confusing.

Reply:
Ah ok that makes sense!